frappe/utils/file_manager.py

```diff
@@ -319,8 +319,10 @@ def get_file_path(file_name):
 	if '../' in file_name:
 		return
 
-	f = frappe.db.sql("""select file_url from `tabFile`
-		where name=%s or file_name=%s""", (file_name, file_name))
+	File = DocType("File")
+
+	f = frappe.qb.from_(File).where((File.name == file_name) & (File.file_name == file_name)).select(File.file_url).run()
```

Comment:
Couldn't this be done using the ORM with `or_filters` instead?

Reply:
Would you find that more readable? I find the behaviour of `or_filters` unpredictable and don't think it should even exist.

Anyway thanks for dropping this, it pointed me to fix this (https://github.com/frappe/frappe/commit/1728db8428654815a4732ccebed93b1517edeec6)

Reply:
A `get_all` with `or_filters` actually seems more readable than a `qb` query, but if there's unpredictability, then it makes sense. I thought the `or_filters` only appends the conditions with a different separator while building the SQL query.